package.json

```diff
@@ -66,10 +66,8 @@
   },
   "scripts": {
     "unit": "jest --testPathPattern=.*.test.js$ --coverage",
-    "e2e": "jest --testPathPattern=.*e2e.* --runInBand",
+    "e2e": "jest --testPathPattern=.*e2e.* --runInBand. --forceExit",
```

Comment:
adding this forceExit is what caused it to actually exit. I tried using [--detectOpenHandles](https://jestjs.io/docs/en/23.x/cli#detectopenhandles) to determine what was wrong (which required the upgrade to 23.x), but it didn't return any results. 

Based on comments on https://github.com/facebook/jest/issues/1456, it seems like it's possible that the issue is not uncommon! and that --runInBand may be the culprit.

I tried running the tests in parallel, but ended up with a lot of salesforce auth errors from trying to sign in at the same time.